harness/attributes/docker-base.yml

```diff
@@ -63,13 +63,39 @@ attributes:
       resources:
         memory: "1024Mi"
     redis:
-      image: redis:5-alpine
+      image: redis:6-alpine
+      options:
```

Comment:
I've started using `config:` as is common in helm charts. What do you think?

Reply:
I know we did start with options in mysql tho

Reply:
or `config.options` to make it more clear it's like cli options but formatted

Reply:
I've made a PR for the mysql options to do this https://github.com/inviqa/harness-docker/pull/74